src/SWP/Component/MultiTenancy/Resolver/TenantResolver.php

```diff
@@ -72,16 +94,19 @@ public function resolve($host = null)
      */
     protected function extractSubdomain($host)
     {
-        if ($this->domain === $host) {
-            return self::DEFAULT_TENANT;
+        $extract = new \LayerShifter\TLDExtract\Extract();
```

Comment:
you can move that part of code to a provate method because you are calling it in two places